tests/integration/test_validate_manifest.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any
+
+import pytest
+
+from schematic.models.validate_manifest import ValidateManifest
+from schematic.schemas.data_model_json_schema import DataModelJSONSchema
+from schematic.utils.df_utils import load_df
+
+
+class TestValidateManifest:
+    """Tests for ValidateManifest class"""
+
+    @pytest.mark.parametrize(
+        ("manifest", "model", "root_node"),
+        [
+            (
+                "mock_manifests/Valid_Test_Manifest_with_nones.csv",
+                "example_test_nones.model.csv",
+                "MockComponent",
+            ),
+            (
+                "mock_manifests/Invalid_Test_Manifest_with_nones.csv",
+                "example_test_nones.model.csv",
+                "MockComponent",
+            ),
+        ],
+    )
+    def test_validate_manifest_values(
+        self, helpers: Any, manifest: str, model: str, root_node: str
+    ):
+        """Tests for ValidateManifest.validate_manifest_values
+
+        Args:
+            helpers (Any): An object with helper functions
+            manifest (str): A path to the manifest to be tested
+            model (str): A path to the model to be tested
+            root_node (str): The name of the component to be tested
+        """
+        # Get manifest and data model path
+        manifest_path: str = helpers.get_data_path(manifest)
+        model_path: str = helpers.get_data_path(model)
+
+        # Gather parameters needed to run validate_manifest_rules
+        errors = []
+        load_args = {
+            "dtype": "string",
+        }
+
+        dmge = helpers.get_data_model_graph_explorer(path=model)
+
+        data_model_js = DataModelJSONSchema(jsonld_path=model_path, graph=dmge.graph)
+        json_schema = data_model_js.get_json_validation_schema(
+            root_node, root_node + "_validation"
+        )
+
+        manifest_df = load_df(
```

Comment:
I'm confused, is there a function within schematic that takes a filepath and loads everything?  I feel like the columns that are lists aren't being read in correctly.  Is that part of `load_df`'s job?

Reply:
> I'm confused, is there a function within schematic that takes a filepath and then validates everything?

Yes, that would be the function you listed. I think @andrewelamb was trying to skip running all the validation rules and just perform the jsonschema validation but I think in this case it's safer to continue to use `validateModelManifest` like we have in `test_validation.py`

Reply:
@andrewelamb going through `validateModelManifest` will be less of an issue with the Biospecimen manifest, there are no validation rules defined, its just the valid values in one attribute.